src/App.tsx

```diff
@@ -1,17 +1,164 @@
-import React from 'react';
-import './App.css';
+import './App.scss';
 
-import users from './api/users';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import TodoList from './components/TodoList/TodoList';
+import { Todo, User, PreparedTodo } from './react-app-env';
+
+const prepare = (
+  users: User[],
+  todos: Todo[],
+): PreparedTodo[] => {
+  return todos.map(todo => ({
+    ...todo,
+    user: users.find(user => user.id === todo.userId) || null,
+  }));
+};
 
 const App: React.FC = () => {
+  const [selectUser, setSelectUser] = useState(0);
+  const [titles, setTitles] = useState('');
+  const [isCompleted, setIsCompleted] = useState(false);
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+  const [todos, setTodos] = useState(todosFromServer);
+  const [user] = useState(usersFromServer);
+  const [counter, setCounter] = useState(2);
```

Comment:
What is 2 as default value in counter? Maybe you should specify the initial length of todos array?